src/utils/bloodglucose.js

```diff
@@ -234,48 +234,15 @@ export function getOutOfRangeThreshold(bgDatum) {
  * @param {Array} data - cgm data
  * @return {Integer} count - the weighted count
  */
-export function weightedCGMCount(data) {
+export function weightedCGMCount(data) { // TODO: do I need to account for 1min samples here?
```

Comment:
Now that each datum has a sampleInterval set, we can use that for the weighted count (currently only used for determining the default bgSource for the trends view)